src/bin/pgcopydb/indexes.c

```diff
@@ -75,6 +75,35 @@ copydb_start_index_supervisor(CopyDataSpec *specs)
 }
 
 
+/*
+ * copydb_copy_not_processed_indexes creates all the indexes that have not been
+ * processed yet.
+ */
+bool
+copydb_copy_not_processed_indexes(CopyDataSpec *specs)
```

Comment:
Name is hard to make sense of, I think. Would prefer something like “copydb_copy_remaining_indexes” maybe, or even “copydb_create_remaining_indexes”.

Reply:
`copydb_create_remaining_indexes` is good., sorry, I'm bad at naming :) Thanks for your suggestion.

Reply:
My main trick is to avoid negation or any kind of logic in the name. Make it obvious. An old source for inspiration for me was http://www.gar1t.com/blog/solving-embarrassingly-obvious-problems-in-erlang.html ; maybe that can be helpful to others too.